registration/src/utils/api.py

```diff
@@ -136,6 +138,19 @@ def get_stats_from_discord(discord_id: str):
 
     return Stats()
 
+def get_mac_addresses_from_shortcode(shortcode: str):
+    res = requests.get(
```

Comment:
generally best to return response obj directly - allows you to handle different status codes differently in different scenarios